### DISCUSSION TITLE
Build 2021.03.30 not working with G6 transmitter

### DISCUSSION THREAD

**Discussion by markmarogil** [Navid200/xDrip#91] [Dexcom]
Hi all,

xDrip build 2021.03.08 showed me an error with transmitter firmware v2.27.2.103, [so per the wiki](https://github.com/Navid200/xDrip/wiki/Firmware-Versions) I installed build 2021.03.30 (pre-release) instead.

Build 2021.03.30 seemingly does not work with any G6 transmitters, as the option to select "G6" as your transmitter defaults to G5. xDrip allows you to select G6, but after ~5-10min of starting the warmup, it gives an error saying the G5 transmitter is not working, even if you selected G6.

Any way to move forward? Should I just use the 03.08 build instead of 03.30?

Edit: [here's the error message saying Data Source is set to G5. ](https://i.imgur.com/RP9F2QY.jpg)It does not register changing it to G6

**Comment by Navid200** [comment]
It should be a settings issue.
Please post a screenshot of the following page:
Settings -> G5/G6 Debug Settings

**Reply by markmarogil** [reply]
Is there a specific setting you're looking for? If it's "G6 Support", that is checked.

The only **un**checked are:

- Restart Sensor
- Fallback to xDrip
- Android 10 workaround
- Scan for G5 constantly
- Force G5 to UI Thread
- Unbond G5 before each read

[Screenshot 1](https://i.imgur.com/G6uxUXK.jpg)
[Screenshot 2](https://i.imgur.com/Dept3xm.jpg)

Thanks for the quick reply.

**Reply by Navid200** [reply]
OK,
Can I see the system status page?

**Reply by markmarogil** [reply]
[Here it is.](https://i.imgur.com/zoh4FPN.jpg)

I never checked this page, and based on it my hypothesis may be wrong since it says "G6 Native".

Maybe I could try forgetting the device and re-adding it? Back to square 1 I suppose.

**Reply by Navid200** [reply]
Would you please swipe left and show that one (G5/G6 Status) too?

**Reply by Navid200** [reply]
You've shown the classic status page.  Can you also show the G5/G6 status page?

**Reply by markmarogil** [reply]
Sorry didn't see it at first

https://i.imgur.com/k8Cdvvx.jpg

**Reply by Navid200** [reply]
Transmitter days: 100.  You cannot use that any longer.  You will need another transmitter:
https://github.com/Navid200/xDrip/wiki/Transmitter-Lifetime

**Reply by markmarogil** [reply]
Ahh, thanks for helping me understand that. Sorry to waste your time

**Reply by Navid200** [reply]
Absolutely no problem.  And thanks for using the discussions.
Please post here (discussions) if you have any questions, need help or suggestions ...
